examples/ex05a/StickModel_accel_base.i

```diff
@@ -488,14 +488,11 @@
 [Executioner]
   type = Transient
   solve_type = 'NEWTON'
-  petsc_options = '-snes_ksp_ew'
-  petsc_options_iname = '-ksp_gmres_restart -pc_type -pc_hypre_type -pc_hypre_boomeramg_max_iter'
-  petsc_options_value = '201                hypre    boomeramg      4'
   end_time = 45.0
   dt = 0.005
   dtmin = 0.001
   nl_abs_tol = 1e-8
-    nl_rel_tol = 1e-06
+  nl_rel_tol = 1e-6
   l_tol = 1e-8
   l_max_its = 20
```

Comment:
Is there a reason why you limited the linear iterations to 20? If not, please remove this too.